tests/test_root_model.py

```diff
@@ -401,3 +401,20 @@ class Model(BaseModel):
 
     m = Model()
     assert m.value.root == 42
+
+
+def test_root_model_json_schema_meta():
+    ParametrizedModel = RootModel[int]
+
+    class SubclassedModel(RootModel):
+        """Subclassed Model docstring"""
+
+        root: int
+
+    parametrized_json_schema = ParametrizedModel.model_json_schema()
+    subclassed_json_schema = SubclassedModel.model_json_schema()
+
+    assert parametrized_json_schema.get('title') == 'RootModel[int]'
```

Comment:
Why is the title not `ParametrizedModel`， but `RootModel[int]`?

```python
ParametrizedModel = RootModel[int]

ParametrizedModel2 = RootModel[int]
```

It would be strange if `ParameterizedModel` and `ParameterizedModel2` had the same title.

Reply:
This is basically an equivalent to

```py
class FirstModel(RootModel):
    root: int

SecondModel = FirstModel

assert SecondModel.model_json_schema()['title'] == 'FirstModel'
```

An assignment `ParametrizedModel = RootModel[int]` does not create a new object it just assigns the value of `RootModel[int]` to the variable `ParametrizedModel`.

Indeed, this test passes

```py
    ParametrizedModel = RootModel[int]
    ParametrizedModel2 = RootModel[int]

    assert ParametrizedModel is ParametrizedModel2
```

Reply:
I'll note we may be able to support this better with `typing_extensions.TypeAliasType`, but @lig is right that without something like that there's no great way to know what the variable name is in python.